ibl_to_nwb/AlyxToNWB/io_tools.py

```diff
@@ -1,24 +1,25 @@
-from lazy_ops import DatasetView
+from datetime import datetime
+from pathlib import PurePath
+
 import numpy as np
 import pandas as pd
-from datetime import datetime
 from tzlocal import get_localzone
-from pathlib import PurePath
 
 
-def iter_datasetview(datasetview_obj):
+def iter_datasetview(datasetview_obj, channel_ids=None):
     """
     Generator to return a row of the array each time it is called.
     This will be wrapped with a DataChunkIterator class.
 
     Parameters
     ----------
-    datasetview_obj: DatasetView
+    datasetview_obj: array-like
         2-D array to iteratively write to nwb.
+    channel_ids: np.array of ints, optional
+        channel numbers to store
     """
-
     for i in range(datasetview_obj.shape[0]//700):
```

Comment:
What is this 7000 for?

Reply:
had added that to test a subset of raw_Datasets, forgot to remove that while commit. Removed now